src/app/qgsapplayertreeviewmenuprovider.cpp

```diff
@@ -56,6 +56,8 @@ QMenu *QgsAppLayerTreeViewMenuProvider::createContextMenu()
   {
     // global menu
     menu->addAction( actions->actionAddGroup( menu ) );
+    menu->addAction( QgsApplication::getThemeIcon( QStringLiteral( "/mActionEditPaste.svg" ) ),
```

Comment:
Can you split the action creation off into its own line so that the connection can use a new style connect please?